.jenkins/projectBuilder.Jenkinsfile

```diff
@@ -48,6 +48,7 @@ projectBuilderV5 (
                 API_BASE_URL: '@vault(secret/configs/upgrade/${environment}/API_BASE_URL)',
                 BASE_HREF_PREFIX: '@vault(secret/configs/upgrade/${environment}/BASE_HREF_PREFIX)',
                 GOOGLE_CLIENT_ID: '@vault(secret/configs/upgrade/${environment}/GOOGLE_CLIENT_ID)',
+                SERVICE_ACCOUNT_ID: '@vault(secret/configs/upgrade/${environment}/SERVICE_ACCOUNT_ID)',
```

Comment:
This value needs to be added to vault so builds dont fail for each env

Reply:
we're still discussing this change @shpwe, it's not something we need to add something in there for yet. @zackcl I included @kcalvo-cli, we should be including SRE review on any infrastructure/config updates, esp right now since it is something being actively worked on.